src/core/importType.js

```diff
@@ -27,7 +27,7 @@ function isExternalModule(name, settings, path) {
   return externalModuleRegExp.test(name) && isExternalPath(path, name, settings)
 }
 
-const scopedRegExp = /^@\w+\/\w+/
```

Comment:
Yes, just tested this on our codebase and that works 👍  I have updated the pull request.